src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.Apps/Utils/ShellCommand.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+
+namespace Microsoft.CmdPal.Ext.Apps.Utils;
+
+public static class ShellCommand
+{
+    public enum RunAsType
+    {
+        None,
+        Administrator,
+        OtherUser,
+    }
+
+    public static ProcessStartInfo GetProcessStartInfo(string target, string parentDir, string programArguments, RunAsType runAs = RunAsType.None)
```

Comment:
ha you know, I think I already pilfered a bunch of this into `ShellHelpers` in the toolkit. I should file a future todo to just de-dupe all the calls that do this everywhere